npc_simulator/src/node.cpp

```diff
@@ -358,37 +370,37 @@ int NPCSimulator::DecideLaneIdWithLaneChangeMode(
     return current_lane_id;
   } else if (obj->lane_change_dir.dir == npc_simulator::msg::LaneChangeDir::LEFT_LANE_CHANGE) {
     if (!checkValidLaneChange(current_lane_id, "left", lane_id)) {
-      //wait lane change until to be possible
+      // wait lane change until to be possible
       return current_lane_id;
     }
 
   } else if (obj->lane_change_dir.dir == npc_simulator::msg::LaneChangeDir::RIGHT_LANE_CHANGE) {
     if (!checkValidLaneChange(current_lane_id, "right", lane_id)) {
-      //wait lane change until to be possible
+      // wait lane change until to be possible
       return current_lane_id;
     }
   } else if (obj->lane_change_dir.dir == npc_simulator::msg::LaneChangeDir::LANE_CHANGE_UTURN) {
     if (!checkValidUTurn(obj->initial_state.pose_covariance.pose, current_lane_id, lane_id)) {
-      //wait u-turn change until to be possible
+      // wait u-turn change until to be possible
       return current_lane_id;
     }
   }
 
   // check existance of lane with target id
   if (!lanelet_map_ptr_->laneletLayer.exists(lane_id)) {
```

Comment:
Can we be sure lane_id is initialized?

Reply:
@tkimura4 Could you answer this question?

Reply:
I changed it to 0 so that the `exists()` call in line 390 will always return false if it hasn't been initialized.